.github/workflows/run-tests.yml

```diff
@@ -34,6 +34,7 @@ jobs:
     name: Run Ember Tests
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
A small change to the workflow to ensure that all ember-try scenarios can run to completion.